frontend/src/service/ShortLink.service.ts

```diff
@@ -0,0 +1,6 @@
+import { Url } from '../entity/Url';
+
+export interface IQueryUrlData {
+  urls: Url[];
+  total: number;
+}
```

Comment:
```suggestion
export interface IPagedShortLinks {
  urls: Url[];
  offset: number;
  totalCount: number;
}
```

Reply:
@byliuyang , do you have any use case in mind for the offset attribute in IPagedShortLinks?

Reply:
@rohithbalaji123 I rethought about it. Actually, no. We don't need `offset` here.